.env.example

```diff
@@ -6,3 +6,6 @@ NODE_ENV=production
 
 # code used to hook up google analytics
 GA_CODE=YOUR_CODE
+
+# url of the current environment
```

Comment:
It should be mentioned somehow that this is only important for a reverse proxy setup. And maybe give an example.